secant/Colors.xcassets/Shadow/primaryButtonShadow.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This looks like it was added by accident, I don't see it being used anywhere.

Reply:
good catch! I tested shadows and looked odd so I removed them. Thanks!